<?xml version="1.0" encoding="UTF-8" ?>

<launch>
    <include file="$(find task_manager_turtlesim)/launch/button_server2.launch"/>
    <node name="turtlesim" pkg="turtlesim" type="turtlesim_node"  />
    <node name="turtlesim_tasks" pkg="task_manager_turtlesim" type="turtlesim_task_server"  output="screen">
        <param name="lib_path" value="$(find task_manager_turtlesim)/tasks"/>
    </node>
</launch>
